docs/resources/condition.md

```diff
@@ -0,0 +1,127 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "lightstep_condition Resource - terraform-provider-lightstep"
+subcategory: ""
+description: |-
+
+---
+
+# lightstep_condition (Resource)
+
+Provides a [Lightstep Condition](https://api-docs.lightstep.com/reference/listalertsid). This can be used to create and manage Lightstep Alerts.
+
+
+## Example Usage
```

Comment:
despite the `generate` comment in this file's header, i had to handwrite this example (based on the existing `lightstep_metric_condition` example). please doublecheck my work here 🙏